### PR TITLE
Feat/create user add response func

### PR DESCRIPTION
**Pull request by delveper**

### CODE REVIEW

File: app/transport/http/const.go [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package http
+
+const (
+	MsgUserCreated      = "user successfully created"
```
Comment (vpakhuchyi) [inline app/transport/http/const.go:4]: I would write msgs that start from capital letter - "User successfully created" etc.

File: app/transport/http/const.go [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+	MsgEmailConflict    = "email is already taken"
+	MsgInternalSeverErr = "internal server error"
+	MsgBadRequest       = "request is bad"
```
Comment (vpakhuchyi) [inline app/transport/http/const.go:7]: Let's not invent the wheel. We have a well known "Bad request"

File: app/transport/http/const.go (hunk 1 above)
Comment (delveper) [reply app/transport/http/const.go:4]: Sure. I forgot about it. @andstepko told us to capitalize these messages.

File: app/transport/http/const.go (hunk 2 above)
Comment (delveper) [reply app/transport/http/const.go:7]: Agreed

File: app/transport/http/const.go (hunk 1 above)
Comment (delveper) [reply app/transport/http/const.go:4]: But what should we do with errors that come from validator?

**Review by vpakhuchyi — APPROVED** [review]

**Review by Brigant — APPROVED** [review]

**Review by lifehou5e — APPROVED** [review]

**Review by YuriiYakovenko — APPROVED** [review]

**Review by Rusli4k — APPROVED** [review]

**Review by vladashakhvorostova — APPROVED** [review]

**Pull request merged**